common/persistence/cassandra/cassandraMetadataPersistenceV2.go

```diff
@@ -573,3 +600,15 @@ func (m *cassandraMetadataPersistenceV2) deleteDomain(name, ID string) error {
 
 	return nil
 }
+
+func (m *cassandraMetadataPersistenceV2) getFailoverConfig(
+	request *p.InternalUpdateDomainRequest,
+) p.DomainFailoverConfig {
+
+	config := p.DomainFailoverConfig{}
+	if request.FailoverConfig != nil {
+		config.StartTime = request.FailoverConfig.StartTime
+		config.Timeout = request.FailoverConfig.Timeout
```

Comment:
On failover config do we want some id to identify which failover this is? I am wondering if I call update domain twice with failover configs if if we want to include some uuid to identify these failover requests in logs. This is mainly for debugability - I don't think its needed for correctness, from a correctness point of view we only care about the latest request.

Reply:
Failover version should be able to tell which failover it is.